lib/webdriver-js-extender/index.js

```diff
@@ -0,0 +1,63 @@
+// Used to provide better protractor documentation for methods given by
+// `webdriver-js-extender`.
+
+/**
+ * @fileoverview Extra methods provided by webdriver-js-extender.
+ */
+
+goog.provide('webdriver_extensions');
+
+// /////////////////////////////////////////////////////////////////////////////
+// //
+// //  webdriver_extensions.ExtendedWebDriver
+// //
+// /////////////////////////////////////////////////////////////////////////////
+/**
+ * Protractor's `browser` object is a wrapper for an instance of
+ * `ExtendedWebDriver`, provided by `webdriver-js-extender`, which itself is
+ * just an instance of `selenium-webdriver`'s WebDriver with some extra methods
+ * added in. The `browser` object inherits all of WebDriver's and
+ * ExtendedWebDriver's methods, but only the methods most useful to Protractor
+ * users are documented here.
```

Comment:
This comment should mention that these are available only if your server has appium.